File: src/ettmotion/api/inference.py
def inference(model, image):
    """
    Performs inference using a given model on an input image.

    Parameters:
        model: The model object used for inference.
        image: The input image on which inference is performed.

    Returns:
        object: The result of the inference process.
    """
    return model.process(image)

File: matlab-api/+smalltargetmotiondetectors/+api/inference.m
function inferResult = inference(model, image)
    % INFERENCE performs inference using a given model on an input image.
    % 
    % Syntax:
    %   inferResult = inference(model, image)
    % 
    % Input:
    %   model - The model object used for inference.
    %   image - The input image on which inference is performed.
    % 
    % Output:
    %   inferResult - The result of the inference process.
    % 
    % Example:
    %   result = inference(myModel, myImage);
    
    inferResult = model.process(image);
end
